Assets/Scripts/Player/WeaponInventory.cs

```diff
@@ -50,6 +53,7 @@ private enum State
 
     private void Start()
     {
+        playerGun = GetComponent<Gun>();

```

Comment:
Would it be beneficial to put this part in an `awake` function instead? Since it's dealing with the player prefab?

Reply:
Yes

Reply:
Actually, nevermind I don't believe awake is needed.

Reply:
👍